lib/Models/Cesium.js

```diff
@@ -792,6 +792,17 @@ Cesium.prototype.pickFromScreenPosition = function(screenPosition) {
 
     var result = this._buildPickedFeatures(providerCoords, pickPosition, vectorFeatures, [pickRasterPromise], undefined, pickPositionCartographic.height);
 
+    if (this.terria.showSplitter) {
+        // Select only features from the same side or both sides of the splitter
+        var side = this.terria.getSplitterSideForScreenPosition(screenPosition);
+        result.allFeaturesAvailablePromise = result.allFeaturesAvailablePromise.then(function() {
+            result.features = result.features.filter(function(feature) {
+                return ( feature.imageryLayer.splitDirection === ImagerySplitDirection.NONE ||
```

Comment:
Not all features have an `imageryLayer` property. If you add the "Bird Migration" layer in TerriaMap, and click on one of the yellow dots, you'll get an exception here.

Because the splitter only applies to imagery layers, I think it might make sense to filter right after the `pickFeatures` call in the `pickFromLocation` function, rather than filtering all results here. Unless you've considered that and it won't work for some reason that I haven't thought of.